packages/frontend/src/features/sqlRunner/store/thunks.ts

```diff
@@ -70,7 +70,7 @@ export const runSqlQuery = createAsyncThunk<
                 return rejectWithValue(job.error);
             }
         } catch (error) {
-            return rejectWithValue(error as ApiErrorDetail);
+            return rejectWithValue(error?.error as ApiErrorDetail);
```

Comment:
error is of type ApiError, not ApiErrorDetail
This was causing the message to be empty 

![Uploading Screenshot from 2024-11-08 10-23-59.png…]()